.circleci/config.yml

```diff
@@ -230,6 +230,12 @@ jobs:
             if [ "$MIGRATE_FLAG" == "true" ]; then
               npm run start:migration -- $ENV
             fi
+      - run:
+          name: Run Postgres Migration
```

Comment:
Rather than running an all colors deploy in order to get a terraform value a suggestion that I have would be to store this value inside of a parameter store like SSM. Then that parameter could be looked up inside of your npm run migration:postgres script.

As an example take a look at web-api/terraform/modules/everything-else-deprecated/failover.tf for storing ssm parameters and web-api/switch-health-check-domain.ts for fetching parameters.